<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1021</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>97</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="Rectangle">
       <property name="text">
        <string>Rectangle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Triangle">
       <property name="text">
        <string>Triangle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Circle">
       <property name="text">
        <string>Circle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Rhombus">
       <property name="text">
        <string>Rhombus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Star_8">
       <property name="text">
        <string>Star_8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Star_6">
       <property name="text">
        <string>Star_6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Star_5">
       <property name="text">
        <string>Star_5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Hexagon">
       <property name="text">
        <string>Hexagon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Special">
       <property name="text">
        <string>Special</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="move_up">
       <property name="text">
        <string>move_up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="move_down">
       <property name="text">
        <string>move_down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="move_left">
       <property name="text">
        <string>move_left</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="move_right">
       <property name="text">
        <string>move_right</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rotate_left">
       <property name="text">
        <string>rotatre_left</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rotate_right">
       <property name="text">
        <string>rotate_right</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="size_plus">
       <property name="text">
        <string>size_plus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="size_minus">
       <property name="text">
        <string>size_minus</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>61</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLineEdit" name="Enter_y_pos"/>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="Enter_size"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>size</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="Perimetr">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="Enter_x_pos"/>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="Enter_angle"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="X_pos">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="Y_pos">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>angle</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="Area">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="angle">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Area</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Perimeter</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="size">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>223</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Rhombus</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Side</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="side"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Smal_angle</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="angle_2"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="create_rhombus">
       <property name="text">
        <string>Create rhombus</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1292</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
